packages/manager/modules/pci/src/projects/project/storages/databases/add/translations/Messages_fr_FR.json

```
@@ -0,0 +1,25 @@
+{
+  "pci_database_add_title": "Créer une base de données",
+  "pci_database_add_description": "Créez une base de données disponible dans votre projet Public Cloud. Si vous souhaitez utiliser un réseau privé et que celui-ci n’existe pas encore, veuillez <a href='{{createPrivateNetworkLink}}'>créer un réseau privé</a> ou un sous-réseau d’abord.",
```

To avoid to insert html snippet code we  can use:
```suggestion
  "pci_database_add_description": "Créer une base de données disponible dans votre projet Public Cloud. Si vous souhaitez utiliser un réseau privé et que celui-ci n’existe pas encore, merci de créer un réseau privé ou un sous-réseau d’abord.",
  "pci_database_add_description_link": "Créer mon réseau."
```

---

It has been separated to sentences